packages/devextreme/js/__internal/grids/new/card_view/content_view/content/card/card.image.tsx

```diff
@@ -0,0 +1,29 @@
+import { PureComponent } from '@ts/grids/new/grid_core/core/pure_component';
+import type { InfernoNode } from 'inferno';
+
+const CLASSES = {
+  cover: 'dx-card-cover',
+};
+
+export interface ImageProps {
+  src?: string;
+  alt?: string;
+  style?: React.CSSProperties;
```

Comment:
I don't see that this `style` prop is used, let's remove it. If it is required to customize image from card component, let's pass `className`